test-engineering/load-tests/load_test_setup.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -eu
+
+#Setup environment
+GCLOUD=$(which gcloud)
+SED=$(which sed)
+KUBECTL=$(which kubectl)
+
+#RED_OUPUT='\033[0;31m'
+DIRECTORY=$(pwd)
+SDK_DIRECTORY=google-cloud-sdk
+MERINO_DIRECTORY=$DIRECTORY/kubernetes-config
+MASTER_FILE=locust-master-controller.yml
+WORKER_FILE=locust-worker-controller.yml
+SERVICE_FILE=locust-master-service.yml
+
+#Declare variables
+LOCUST_IMAGE_TAG=$(git log -1 --pretty=format:%h)
+#LOCUST_IMAGE_TAG=qwe123
+echo "Docker image tag for locust is set to: ${LOCUST_IMAGE_TAG}"
+
+##Kubernetes Manifests variables
+KINTO_SERVER_URL=firefox.settings.services.mozilla.com
+KINTO_COLLECTION=quicksuggest
+KINTO_BUCKET=main
+CLUSTER=merino-load-test
+TARGET=stage.merino.nonprod.cloudops.mozgcp.net
+SCOPE="https://www.googleapis.com/auth/cloud-platform"
+REGION=us-central1
+ZONE=${REGION}-b
+PROJECT=spheric-keel-331521
+
+#Configure Kubernetes
+echo -e "==================== Prepare environments with set of environment variables "
+echo -e "==================== Set Kubernetes Cluster "
+export CLUSTER=$CLUSTER
+echo -e "==================== Set Kubernetes TARGET "
+export TARGET=$TARGET
+echo -e "==================== Set SCOPE "
+export SCOPE=$SCOPE
+
+echo -e "==================== Prepare GCP by updating default zone and project ID "
+$GCLOUD config set compute/zone $ZONE
+$GCLOUD config set project $PROJECT
```

Comment:
Are these two steps required on Cloud Shell?

Reply:
After testing looks like it's not. So removed it. Thanks for the catch!